bundles/org.openhab.io.metrics/src/main/java/org/openhab/io/metrics/MetricsRestController.java

```diff
@@ -79,5 +91,29 @@ public void setMeterRegistryProvider(MeterRegistryProvider meterRegistryProvider
         meterRegistry = meterRegistryProvider.getOHMeterRegistry();
         Objects.requireNonNull(meterRegistry).add(prometheusMeterRegistry);
         logger.debug("Core metrics registry retrieved and Prometheus registry added successfully.");
+        updateMeterRegistry();
+    }
+
+    @Activate
+    protected void activate(Map<@Nullable String, @Nullable Object> configuration) {
+        logger.info("Metrics service activated.");
```

Comment:
The previous log was mentioning the path where the metrics are available - that was helpful and was ok to log as info. Without it, I'd rather suggest to just do debug. Up to you, which option to choose.